### ISSUE TITLE
Do not "nudge" the chat when the pause option is enabled

### ISSUE THREAD

**Issue by kuon** [Durss/Twitchat#68]
When the "pause on hove" option is enabled, the chat is nudged toward the top of about 20 pixels when hovering the chat.

**Comment by Durss** [comment]
Isn't it because of that "chat paused" appearing at the bottom ?
![image](https://github.com/Durss/Twitchat/assets/721001/2b89ceff-e9ae-4bca-9e8b-209373029420)

**Comment by kuon** [comment]
Yes it is because of the chat paused banner, but I think the banner should not alter the layout, otherwise it is hard to aim when moderating.
